test/pool-bin/libraries/BinHelper.t.sol

```diff
@@ -59,6 +59,22 @@ contract BinHelperTest is BinTestHelper {
         }
     }
 
+    // run test via: forge test --match-test testLiquidityPOC -vvv
+    function testLiquidityPOC() external {
```

Comment:
Can set pure , or will get warning.
![Screenshot 2024-07-18 at 13 05 15](https://github.com/user-attachments/assets/91a5d67a-e981-48d5-9828-91d013a2ed34)

Reply:
Just noticed this is a POC 😂